packaging/ansible-runner-service-project/project/roles/gluster-replace-peers/tasks/peers.yml

```diff
@@ -2,49 +2,48 @@
 - name: Validating the state of the old host
   block:
     - name: Fetch the UUID of the old node
-      shell: >
+      ansible.builtin.shell: >
         gluster peer status |
         grep -A1 {{ gluster_old_node | mandatory }} |
         awk -F: '/uid/ { print $2}'
       register: uuid
-      tags:
-        - skip_ansible_lint # E301
 
     - name: Get the state from peer file
-      shell: grep state "/var/lib/glusterd/peers/{{ uuid.stdout | trim }}"
+      ansible.builtin.command:
+        argv:
+          - grep
+          - state
+          - "/var/lib/glusterd/peers/{{ uuid.stdout | trim }}"
       register: grepres1
-      tags:
-        - skip_ansible_lint # E301
 
     - name: Get the state from the peer command
-      shell: >
+      ansible.builtin.shell: >
         gluster peer status |
         grep -A1 {{ uuid.stdout | trim }} |
         grep Connected |
         wc -l
       ignore_errors: true
       register: grepres2
-      tags:
-        - skip_ansible_lint # E301
 
     - name: Fail the play if the previous command did not succeed
-      fail:
+      ansible.builtin.fail:
         msg: "{{ gluster_old_node }} is already in connected state"
       when: grepres1.stdout == "state=3" and grepres2.stdout == "1"
   run_once: true
 
 - name: Get the UUID of the old node
-  shell: >
+  ansible.builtin.shell: >
+    set -o pipefail ;
```

Comment:
@gobindadas Could `gluster peer status` return non-zero return code, which wouldn't be considered as an error?

Reply:
right. I reverted and change and ignoring lint for this task - https://github.com/oVirt/ovirt-engine/pull/684/commits/ae7af128902429fa91c371c3b2c54b90f3209131

Reply:
@mwperina Nope, if no peers are available then it will return 0